### PR TITLE
Address review concerns on security test evidence.

### PR DESCRIPTION
**Pull request by skisel-da**
CHANGELOG_BEGIN
CHANGELOG_END

### CODE REVIEW

File: ledger/sandbox-on-x/src/test/lib/scala/com/daml/platform/sandbox/auth/AdminServiceCallAuthTests.scala [hunk 1]
```diff
@@ -63,40 +64,42 @@ trait AdminServiceCallAuthTests extends SecuredServiceCallAuthTests {
       )
     )
   }
-  it should "deny calls with freshly created non-admin user" taggedAs securityAsset.setAttack(
-    attack(threat = "Exploit a freshly created non-admin user")
+  it should "deny calls with freshly created non-admin user" taggedAs adminSecurityAsset.setAttack(
+    attackPermissionDenied(threat = "Present a freshly created non-admin user")
```
Comment (soren-da) [inline ledger/sandbox-on-x/src/test/lib/scala/com/daml/platform/sandbox/auth/AdminServiceCallAuthTests.scala:68]:
```suggestion
    attackPermissionDenied(threat = "Present a user JWT for a freshly created non-admin user")
```

File: ledger/sandbox-on-x/src/test/lib/scala/com/daml/platform/sandbox/auth/MultiPartyServiceCallAuthTests.scala [hunk 2]
```diff
@@ -104,7 +104,9 @@ trait MultiPartyServiceCallAuthTests extends SecuredServiceCallAuthTests {
 
   it should "deny single-party calls authorized to no parties (party)" taggedAs securityAsset
     .setAttack(
-      attack(threat = "Exploit a single-party call authorized to no parties (party)")
+      attackPermissionDenied(threat =
+        "Exploit a single-party call authorized to no parties (party)"
```
Comment (soren-da) [inline ledger/sandbox-on-x/src/test/lib/scala/com/daml/platform/sandbox/auth/MultiPartyServiceCallAuthTests.scala:108]:
```suggestion
        "Execute a single-party call authorized to no parties (party)"
```

as `exploit` is very generic again.

If you decide to accept this suggestion please also change the other occurrences below.

File: ledger/sandbox-on-x/src/test/lib/scala/com/daml/platform/sandbox/auth/ServiceCallAuthTests.scala [hunk 3]
```diff
@@ -32,12 +32,27 @@ trait ServiceCallAuthTests
...
+    SecurityTest(property = Authorization, asset = s"User Endpoint $serviceCallName")
 
-  def attack(threat: String): Attack = Attack(
+  val adminSecurityAsset: SecurityTest =
+    SecurityTest(property = Authorization, asset = s"Admin Endpoint $serviceCallName")
+
+  def attackPermissionDenied(threat: String): Attack = Attack(
+    actor = s"Ledger API client calling $serviceCallName",
+    threat = threat,
+    mitigation = s"Refuse to connect the user with PERMISSION_DENIED to $serviceCallName",
```
Comment (soren-da) [inline ledger/sandbox-on-x/src/test/lib/scala/com/daml/platform/sandbox/auth/ServiceCallAuthTests.scala:43]:
```suggestion
    mitigation = s"Refuse call by the client with PERMISSION_DENIED to $serviceCallName",
```

File: ledger/sandbox-on-x/src/test/lib/scala/com/daml/platform/sandbox/auth/ServiceCallAuthTests.scala [hunk 4]
```diff
@@ -32,12 +32,27 @@ trait ServiceCallAuthTests
...
+  )
+
+  def attackInvalidArgument(threat: String): Attack = Attack(
+    actor = s"Ledger API client calling $serviceCallName",
+    threat = threat,
+    mitigation = s"Refuse to connect the user with INVALID_ARGUMENT to $serviceCallName",
```
Comment (soren-da) [inline ledger/sandbox-on-x/src/test/lib/scala/com/daml/platform/sandbox/auth/ServiceCallAuthTests.scala:49]:
```suggestion
    mitigation = s"Refuse call by the client with INVALID_ARGUMENT to $serviceCallName",
```

File: ledger/sandbox-on-x/src/test/lib/scala/com/daml/platform/sandbox/auth/ServiceCallAuthTests.scala [hunk 5]
```diff
@@ -32,12 +32,27 @@ trait ServiceCallAuthTests
...
+  )
+
+  def attackUnauthenticated(threat: String): Attack = Attack(
     actor = s"Ledger API client calling $serviceCallName",
     threat = threat,
-    mitigation = s"Refuse to connect the user to $serviceCallName",
+    mitigation = s"Refuse to connect the user with UNAUTHENTICATED to $serviceCallName",
```
Comment (soren-da) [inline ledger/sandbox-on-x/src/test/lib/scala/com/daml/platform/sandbox/auth/ServiceCallAuthTests.scala:55]:
```suggestion
    mitigation = s"Refuse call by the client with UNAUTHENTICATED to $serviceCallName",
```

**Review by soren-da — APPROVED** [review]
Thanks for doing the clarifications

File: ledger/sandbox-on-x/src/test/lib/scala/com/daml/platform/sandbox/auth/AdminServiceCallAuthTests.scala [hunk 6]
```diff
@@ -17,44 +17,45 @@ trait AdminServiceCallAuthTests extends SecuredServiceCallAuthTests {
...
-    attack(threat = "Exploit an expired admin token")
+  it should "deny calls with an expired admin token" taggedAs adminSecurityAsset.setAttack(
+    attackUnauthenticated(threat = "Present an expired admin JWT")
   ) in {
     expectUnauthenticated(serviceCallWithToken(canReadAsAdminExpired))
   }
-  it should "deny calls with a read-only token" taggedAs securityAsset.setAttack(
-    attack(threat = "Exploit a read-only token for a random party")
+  it should "deny calls with a read-only token" taggedAs adminSecurityAsset.setAttack(
+    attackPermissionDenied(threat = "Present a read-only user JWT for an unknown party")
```
[the remark this answers is not in the document]
Comment (skisel-da) [reply ledger/sandbox-on-x/src/test/lib/scala/com/daml/platform/sandbox/auth/AdminServiceCallAuthTests.scala:31]: yes, rephrased a bit

File: ledger/sandbox-on-x/src/test/lib/scala/com/daml/platform/sandbox/auth/MultiPartyServiceCallAuthTests.scala (hunk 2 above)
Comment (skisel-da) [reply ledger/sandbox-on-x/src/test/lib/scala/com/daml/platform/sandbox/auth/MultiPartyServiceCallAuthTests.scala:108]: ah missed, this file, will fix

**Pull request merged**